TO BE MERGED FOR MTV 2.8.1: MTV 2.8.1 Release notes (#673)
kubev2v/forklift-documentation: documentation/modules/rn-28-resolved-issues.adoc

{project-first} 2.8 has the following resolved issues:

[id="resolved-issues-2-8-1_{context}"]
== Resolved issues 2.8.1

> .VMs with mututally exclsuive spec values as source VNd and destination cannot be migrated

Reviewer: ```suggestion
.VMs with mutually exclusive spec values as source VNd and destination cannot be migrated
```

Reviewer: Are you sure about VNd... I cannot see ` Virtual Network Device (vnd)` mentioned in the JIRA

> In earlier releases of {project-short}, it was possible for a VM to have the `spec` value `Running` as a source VM and have the `spec` value `runStrategy` as a destination VM. This caused migrations to fail because `Running` and `runStrategy` are mutually exclusive. This issue has been resolved in {project-short} 2.8.1. link: https://issues.redhat.com/browse/MTV-2092[(MTV-2092)]

Reviewer: ```suggestion
In earlier releases of {project-short}, it was possible for a VM to have the `spec` value `Running` as a source VM with the `spec` value `runStrategy` as a destination VM. This caused migrations to fail because `Running` and `runStrategy` are mutually exclusive. This issue has been resolved in {project-short} 2.8.1. link: https://issues.redhat.com/browse/MTV-2092[(MTV-2092)]
```

> .The code base of MTV contains a depreacted value for a parameter

Reviewer: ```suggestion
.The code base of MTV contains a deprecated value for a parameter
```

In earlier releases of {project-short}, the code base contained the deprecated value `vm.Spec.Running`, which caused some migrations to fail. This issue has been resolved in {project-short} 2.8.1. link: https://issues.redhat.com/browse/MTV-2105[(MTV-2105)]

.Migrations using custom teplates fail if a user-supplied name is too long

> In earlier releases of {project-short},migrations that used custom templates failed if a user-supplied was too long because it exceeded 63 characters. This issue has been resolved in {project-short} 2.8.1. Now {project-short} automatically truncates names that are too long. link: https://issues.redhat.com/browse/MTV-2244[(MTV-2244)]

Reviewer: ```suggestion
In earlier releases of {project-short}, migrations that used custom templates failed if a user-supplied was too long because it exceeded 63 characters. This issue has been resolved in {project-short} 2.8.1. Now {project-short} automatically truncates names that are too long. link: https://issues.redhat.com/browse/MTV-2244[(MTV-2244)]
```

.User supplied template names are not validated against the RFC DNS1123 standard
In earlier releases of {project-short}, user-supplied template names were not validated against the RFC DNS1123 standard. This caused migrations of VMs with template names that did not conform to RFC DNS1132 to fail. This issue has been resolved in {project-short} 2.8.1. link: https://issues.redhat.com/browse/MTV-2249[(MTV-2249)]

> .Cancelling the migration of a virtual machine causes the Plan controller to loop forever

Reviewer: ```suggestion
.Canceling the migration of a virtual machine causes the Plan controller to loop forever
```

Reviewer: US spelling

In earlier releases of {project-short}, if the migration of a virtual machine (VM) was canceled, the scheduler of the Plan controller would endlessly reschedule it for migration. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2287[(MTV-2287)]

.MTV fails to migrate VMs with template names that contain multiple periods

> In earlier releases of {project-short}, a VM whose name contained multiple periods (for example, `vm...` ) could not be migrated because its name did not comply with the application and support protocol RFC DNS1123. This issue has been resolved in {project-short} 2.8.1. Now, {project-short} automatically changes multiple periods in a VM name to a single period, making it compliant with RFC DNS1123. link:https://issues.redhat.com/browse/MTV-2308[(MTV-2308)]

Reviewer: ```suggestion
In earlier releases of {project-short}, a VM whose name contained multiple periods (for example, `vm...`) could not be migrated because its name did not comply with the application and support protocol RFC DNS1123. This issue has been resolved in {project-short} 2.8.1. Now, {project-short} automatically changes multiple periods in a VM name to a single period, making it compliant with RFC DNS1123. link:https://issues.redhat.com/browse/MTV-2308[(MTV-2308)]
```

> In earlier releases of {project-short}, a VM whose name contained multiple periods (for example, `vm...` ) could not be migrated because its name did not comply with the application and support protocal RFC DNS1123. This issue has been resolved in {project-short} 2.8.1. Now, {project-short} automatically changes multiple periods in a VM name to a single period, making the compliant with RFC DNS1123. link: https://issues.redhat.com/browse/MTV-2308[(MTV-2308)]

Reviewer: ```suggestion
In earlier releases of {project-short}, a VM whose name contained multiple periods (for example, `vm...` ) could not be migrated because its name did not comply with the application and support protocol RFC DNS1123. This issue has been resolved in {project-short} 2.8.1. Now, {project-short} automatically changes multiple periods in a VM name to a single period, making it compliant with RFC DNS1123. link: https://issues.redhat.com/browse/MTV-2308[(MTV-2308)]
```

> .UI displays incorrect "Invalid VM Name" message for a VM name containing a period

Reviewer: ```suggestion
.UI displays an incorrect "Invalid VM Name" message for a VM name containing a period
```

> In earlier releases of {project-short}, the UI displayed an "Invalid VM Name" message for a VM name containing a period, even though {project-short} supports such names. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2309[(MTV-2309)]

Reviewer: ```suggestion
In earlier releases of {project-short}, the UI displayed an "Invalid VM Name" message for a VM name containing a period, even though {project-short} supports such names. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2309[(MTV-2309)]
```

.The code base of MTV contains a deprecated value for a parameter
In earlier releases of {project-short}, the code base contained the deprecated value `vm.Spec.Running`, which caused some migrations to fail. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2105[(MTV-2105)]

.Migrations using custom templates fail if a user-supplied name is too long
In earlier releases of {project-short}, migrations that used custom templates failed if a user-supplied was too long because it exceeded 63 characters. This issue has been resolved in {project-short} 2.8.1. Now {project-short} automatically truncates names that are too long. link:https://issues.redhat.com/browse/MTV-2244[(MTV-2244)]

.User supplied template names are not validated against the RFC DNS1123 standard
In earlier releases of {project-short}, user-supplied template names were not validated against the RFC DNS1123 standard. This caused migrations of VMs with template names that did not conform to RFC DNS1132 to fail. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2249[(MTV-2249)]

.Disk names are changed after migration

> In earlier releases of {project-short}, migrating a VM that had disks with different buses resulted in disk names being changed after migration. This issue has been resolved in {project-short} 2.8.1. link:https://issues.redhat.com/browse/MTV-2266[(MTV-2266)]

Reviewer 2: Should be https://issues.redhat.com/browse/MTV-2262 but we found issues with the fix and moving it out from 2.8.1. Please remove this part
